modules/test/playwright/helpers/ObjectAdminApiHelper.ts

```diff
@@ -119,6 +119,40 @@ export class ObjectAdminApiHelper {
 		);
 	}
 
+	async postFormulaObjectField(
```

Comment:
@davisaints does this need to be alphabetized?  I see other function names tend to follow that pattern, but `postIntegerObjectField` is also out of place

Reply:
@brianwulbern, yes, it must be in alphabetical order. I had considered this and thought that the source format would handle it, as it does in the `.testcase` and `.macro` files. I did it in [e7cd9](https://github.com/brianwulbern/liferay-portal/pull/437/commits/e7cd94f23e84083468fef76453e0641500939973).